File: API/Entities/Campground.cs
namespace API.Entities
{
    public class Campground
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Image { get; set; }
        public string Price { get; set; } 
        public string Description { get; set; }
        public string Location { get; set; }
        public ICollection<Review> Reviews { get; set; }
        public int UserId { get; set; }
        public User User { get; set; }
        public ICollection<Photo> Photos { get; set; } = new List<Photo>();
    }
}

File: client/src/app/_models/campground.ts
import { User } from "./User";
import { Review } from "./review";

export interface Campground {
    id: number;
    title: string;
    price: string;
    description: string;
    location: string;
    image: string;

    reviews: Review[];
    user: User;
    userId: number;
}